src/sentry/api/urls.py

```diff
@@ -3265,6 +3266,12 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         SecretScanningGitHubEndpoint.as_view(),
         name="sentry-api-0-secret-scanning-github",
     ),
+    # Tempest
+    re_path(
+        r"^(?P<organization_id_or_slug>[^\/]+)/(?P<project_id_or_slug>[^\/]+)/tempest-credentials/$",
+        TempestCredentialsEndpoint.as_view(),
+        name="sentry-api-0-project-tempest-credentials",
+    ),
```

Comment:
As this is a ProjectEndpoint, we should probably put this routing rule into the `PROJECT_URLS` constant in this file to place it with the other project endpoints, or are we not doing this on purpose?

Reply:
Uuups, my bad, wanted to add it to the end of the list, but by accident added it to the end of the file 😬